recipe-server/Dockerfile

```diff
@@ -13,14 +13,18 @@ RUN curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add -
     apt-get update && apt-get install -y nodejs
 RUN npm install -g npm@5
 
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
```

Comment:
The usage of `tee` in the docs was likely to make sudo usage possible. Since we don't need sudo, it would probably be best to do this the same way we do the nodesource repositories above, with simple shell redirection.

It would also be good to add this to the `RUN` command on line 10, so we don't add more steps to the docker file (which makes builds slower and the Docker image's file size larger)